usecase/taxi/greenAndYellowUrls.txt

```diff
@@ -0,0 +1,125 @@
+https://s3.amazonaws.com/nyc-tlc/trip+data/green_tripdata_2013-08.csv
```

Comment:
yea it is - that's all I've been using for the demo, and the count queries come back at 1.2b or something